### PR TITLE
Aliases Ctrl+c Exit fix

### LINKED ISSUE #18: [CLI] Selector menu not getting cancelled
### Current:
The selector menu blocks the main thread, and CTRL+C does not cancel the menu, nor exits the program.

### Expectation:
Pressing CTRL+C should clear the menu and exit the program gracefully.

### PR DESCRIPTION
**Pull request by gjyotin305**
Closes #18

### CODE REVIEW

File: internal/pkg/selector/selector.go [hunk 1]
```diff
@@ -88,6 +89,8 @@ func (m *Menu) Display() string {
 		} else if keyCode == down {
 			m.CursorPos = (m.CursorPos + 1) % len(m.MenuItems)
 			m.renderMenuItems(true)
+		} else if keyCode == ctrl_c {
+			return "Exited !"
```
Comment (SaahilNotSahil) [inline internal/pkg/selector/selector.go:93]: Return a blank string, and add checks wherever this function is called.

**Review by SaahilNotSahil — CHANGES_REQUESTED** [review]
Please make the requested change.

File: internal/pkg/selector/selector.go (hunk 1 above)
Comment (gjyotin305) [reply internal/pkg/selector/selector.go:93]: ok

**Comment by gjyotin305** [comment]
@SaahilNotSahil Please check and review

**Review by SaahilNotSahil — APPROVED** [review]
LGTM

**Pull request merged**